components/application/logout.jsx

```diff
@@ -7,7 +7,7 @@ import { withGlobalStore } from 'store'
 
 const LOGOUT_URL = `https://api.dev.core.ac.uk/logout?continue=${encodeURI(
   typeof window !== 'undefined'
-    ? `${window.location.protocol}//${window.location.host}`
+    ? `${window.location.protocol}//${window.location.host}/logout.html`
```

Comment:
You may find useful [`window.location.origin`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/origin)